index.html

```diff
@@ -566,7 +574,12 @@
 
 In computer systems and on the Web, an [=identity=] seen by a particular website is typically
 assigned an <dfn>identifier</dfn> of some type, which makes it easier for an automated system to
-store data about that [=person=].
+store data about that [=person=]. Examples of [=identifiers=] for a [=person=] can be their name,
+an identification number including those mapping to a device that this [=person=] may be using, their
+phone number, their location data, an online identifier such as email or IP addresses, or
+factors specific to their physical, physiological, genetic, mental, economic, cultural, or social
+[=identity=]. Strings derived from [=identifiers=], for instance through hashing, are still
+[=identifiers=] so long as they may identify a [=person=].
```

Comment:
Bulleted lists are easier to read than long sentences:

```suggestion
store data about that [=person=]. Examples of [=identifiers=] for a [=person=] can be:
* their name,
* an identification number including those mapping to a device that this [=person=] may be using,
* their phone number,
* their location data,
* an online identifier such as email or IP addresses, or
* factors specific to their physical, physiological, genetic, mental, economic, cultural, or social
[=identity=].

Strings derived from [=identifiers=], for instance through hashing, are still
[=identifiers=] so long as they may identify a [=person=].
```